### ISSUE TITLE
done update image other repo 

### ISSUE THREAD

**Issue by vnguyentv2** [silinternational/ecs-deploy#263]
i have current task with image:  "image": "vegacandle/fargate-poc-03:v2",
when i using ecs-deploy with cli  ecs-deploy -c development-ecs-fargate-shared -d development--api-booking -i xxxxxxxx.dkr.ecr.ap-southeast-1.amazonaws.com/api-booking:development-6a90411a5b03fb47115dafe01cfb11ad3e674fa9 --enable-rollback -t 720

it not change image. how can fix it

**Comment by helle253** [comment]
I just ran into this issue. It seems like switching to different repositories isn't handled gracefully, but there is a workaround.

You'll need to create a new task revision, with an image in the repository you are changing to.

Once this is done, subsequent runs of this script SHOULD start using the right image repository, and update the image as expected.

**Comment by devon-sil** [comment]
Known issue, closing.  #105, #238
